<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConditionEdit</class>
 <widget class="QWidget" name="ConditionEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>34</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QComboBox" name="operationCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>&lt; не выбрано &gt;</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>от ... до ...</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>больше</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>меньше</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>равно</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>не равно</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>цифрокод</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="operandASpin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="operandBSpin">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-100000.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="operandText"/>
   </item>
   <item>
    <widget class="QPushButton" name="okBtn">
     <property name="toolTip">
      <string>Принять</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../etc/meteo.icons.qrc">
       <normaloff>:/meteo/icons/ok.png</normaloff>:/meteo/icons/ok.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelBtn">
     <property name="toolTip">
      <string>Отмена</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../etc/meteo.icons.qrc">
       <normaloff>:/meteo/icons/close.png</normaloff>:/meteo/icons/close.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../etc/meteo.icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>ConditionEdit</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
